README.md

```diff
@@ -109,7 +108,9 @@ require additional configurations applied to
 
 The following prerequisites are required to use the helm chart:
 
-- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported)
+- [Helm 3](https://helm.sh/docs/intro/install/)
+  - Helm 3 version support may vary depending on the [supported Kubernetes version](#supported-kubernetes-distributions) you are using and the [Helm supported Kubernetes versions skew](https://helm.sh/docs/topics/version_skew/#supported-version-skew).
```

Comment:
Would be helpful to state we use 3.11 in maintaining this project for some frame of reference.

Reply:
I do want this, but I want it to update the value automatically as needed. I'll see if we can make this a badge.

Reply:
Only added a static reference for now. Will try to add automation later for this in another ticket.